webpack.config.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+    entry: ['babel-polyfill', './parser.js'],
+    output: {
+        path: './bin',
+        filename: 'page-metadata-parser.bundle.js',
```

Comment:
And I'm guessing/hoping that this will fail our linter big time.
We'll probably want to add "bin/*.bundle.js" (or full version) to the .eslintignore file so ESLint ignores this generated/minified/obfusticated file.